src/docfx/build/legacy/LegacyUtility.cs

```diff
@@ -27,9 +27,9 @@ public static string ToLegacyPathRelativeToBasePath(this Document doc, Docset do
             return PathUtility.NormalizeFile(Path.GetRelativePath(docset.Config.DocumentId.SourceBasePath, doc.FilePath));
         }
 
-        public static string ToLegacyOutputPathRelativeToBaseSitePath(this Document doc, Docset docset)
+        public static string ToLegacyOutputPathRelativeToSiteBasePath(this Document doc, Docset docset, string outputFilePathRelativeToSiteBasePath)
         {
-            var legacyOutputFilePathRelativeToSiteBasePath = doc.SitePath;
+            var legacyOutputFilePathRelativeToSiteBasePath = outputFilePathRelativeToSiteBasePath;
```

Comment:
we already did the trim 'SiteBasePath' here, we need do it again?

Reply:
we haven't, should rename this variable to avoid confusion

---
In reply to: [284136194](https://github.com/dotnet/docfx/pull/4540#discussion_r284136194) [](ancestors = 284136194)